releases/SN30_Pro+/3.02/sn30pro_plus.metainfo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2019 Mario Limonciello <mario.limonciello@dell.com> -->
+<component type="firmware">
+  <id>com.8bitdo.sn30pro_plus.firmware</id>
+  <name>SN30 Pro+</name>
+  <summary>Firmware for the 8Bitdo SN30 Pro+ Game Controller</summary>
+  <description>
+    <p>
+      Updating the firmware on your SN30 Pro+ device improves performance and adds
+      new features.
+    </p>
+  </description>
+  <provides>
+    <!-- USB\VID_2DC8&PID_5750&REV_0200 -->
+    <firmware type="flashed">78acf201-a624-5c0b-a521-e204be14e3d5</firmware>
+    <!-- USB\VID_2DC8&PID_5750 -->
```

Comment:
https://github.com/fwupd/8bitdo-firmware/issues/25 was reported indicating the wrong GUID is used here.

This shouldn't be the bootloader GUID, it should be the device GUID, so these:
```
│                          64eb3e9c-e79c-50d0-984b-21a1641f37c5 ← USB\VID_2DC8&PID_6002&REV_0001
│                          a9a82ee0-7fdc-5d80-bd73-6b39f7a4f061 ← USB\VID_2DC8&PID_6002
```

Reply:
Left a response in #25.

Reply:
Done.